x/deployment/keeper/key.go

```diff
@@ -17,7 +17,7 @@ var (
 
 func deploymentKey(id types.DeploymentID) []byte {
 	buf := bytes.NewBuffer(deploymentPrefix)
-	buf.Write(id.Owner.Bytes())
+	buf.Write([]byte(id.Owner))
```

Comment:
This really increases the size of our indexes.

We should look at using `AccAddress` instead.  It's not necessary for this PR tho.